src/tests/ftest/util/dfuse_utils.py

```diff
@@ -273,7 +274,11 @@ def run(self, check=True):
         self.create_mount_point()
 
         # run dfuse command
-        cmd = "".join([self.env.get_export_str(), self.__str__()])
+        cmd = self.env.get_export_str()
+        if bind_cores:
+            cmd += 'taskset -c {} '.format(bind_cores)
+        cmd += str(self)
+        self.log.info("Command is '{}'".format(cmd))
```

Comment:
(pylint-logging-format-interpolation) Use lazy % formatting in logging functions